index.bs

```diff
@@ -532,14 +532,15 @@ spec: workers; urlPrefix: https://www.w3.org/TR/workers/#
     <pre class='idl'>
       dictionary MediaCapabilitiesDecodingInfo : MediaCapabilitiesInfo {
         required MediaKeySystemAccess keySystemAccess;
+        MediaDecodingConfiguration configuration;
       };
     </pre>
 
-    <p>
-      The {{MediaCapabilitiesInfo}} has an associated <dfn
-      for='MediaCapabilitiesInfo'>configuration</dfn> which is a
-      {{MediaDecodingConfiguration}} or {{MediaEncodingConfiguration}}.
-    </p>
+    <pre class='idl'>
+      dictionary MediaCapabilitiesEncodingInfo : MediaCapabilitiesInfo {
+        MediaEncodingConfiguration configuration;
```

Comment:
Since this is defined for all descendents of MediaCapabilitiesInfo, could we move it to the parent dict?

Reply:
I think we're ready to once ^^ this minor question is settled :).

Reply:
Oh, I don't think we can move it to the parent, because of IDL. MediaEncodingConfiguration and MediaDecodingConfiguration are disjoint classes, and moving it to the parent would require that parameter be an "any".

Reply:
Ahh, good point!